### PR TITLE
Special systems agent's installation improvements

### LINKED ISSUE #5928: Add note about the need for administrator privileges during Solaris and HP-UX agent installation
# Description

Currently, there's no note/hint about the need for administrator privileges during agent installation for Solaris and HP-UX agents.

- https://documentation-dev.wazuh.com/4.4/installation-guide/wazuh-agent/wazuh-agent-package-hpux.html
- https://documentation-dev.wazuh.com/4.4//installation-guide/wazuh-agent/wazuh-agent-package-solaris.html

This information already exists on agent installation guides for Linux, macOS, and Windows.

image

- https://documentation-dev.wazuh.com/4.4//installation-guide/wazuh-agent/wazuh-agent-package-linux.html
- https://documentation-dev.wazuh.com/4.4/installation-guide/wazuh-agent/wazuh-agent-package-windows.html
- https://documentation-dev.wazuh.com/4.4/installation-guide/wazuh-agent/wazuh-agent-package-macos.html

### PR DESCRIPTION
**Pull request by jnasselle**
## Description
Closes #5928, #5929 and #5930

- #5928 and #5929
image
image

- #5928 and #5930
image

## Checks
- [x] Compiles without warnings.
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst [hunk 1]
```diff
@@ -12,38 +12,23 @@ The installed agent runs on the endpoint you want to monitor and communicates wi
 
 #. To start the installation process, download the `HP-UX installer <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR_HPUX|/hp-ux/wazuh-agent-|WAZUH_CURRENT_HPUX|-|WAZUH_REVISION_HPUX|-hpux-11v3-ia64.tar>`_. 
 
+.. note:: You need root user privileges to run all the commands described below.
+
```
Comment (rauldpm) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:16]: Fix list numbers (there is two number 1), also, the `Note` should be before the "To start the installation..."

image

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst [hunk 2]
```diff
@@ -12,38 +12,23 @@ The installed agent runs on the endpoint you want to monitor and communicates wi
...
-   
-      <server>
-        <address>WAZUH_MANAGER_IP</address>
-        <port>1514</port>
-        <protocol>tcp</protocol>
-      </server>
-
-
-#. To complete the installation and registration process, start the Wazuh agent service.
+#. Uncompress the package in ``/``.
```
Comment (rauldpm) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:24]: I think "Unzip" should be valid too

- https://www.merriam-webster.com/dictionary/uncompress

File: source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst [hunk 3]
```diff
@@ -14,6 +14,8 @@ The deployment of a Wazuh agent on an AIX system uses deployment variables that
 
 #. To start the deployment process, download the `AIX installer <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR_AIX|/aix/wazuh-agent-|WAZUH_CURRENT_AIX|-|WAZUH_REVISION_AIX|.aix.ppc.rpm>`_. 
 
+.. note:: You need root user privileges to run all the commands described below.
+
```
Comment (rauldpm) [inline source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst:18]: Can you add this to the AIX deployment too?

**Review by rauldpm — CHANGES_REQUESTED** [review]
Nice, minor changes

**Review by rauldpm — CHANGES_REQUESTED** [review]
I am sorry, this comment was not sent

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst (hunk 1 above)
Comment (jnasselle) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:16]: The idea was to reduce the scope of `root` requirement

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst (hunk 2 above)
Comment (jnasselle) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:24]: Yes, is valid too. But uncompress is more accurate

File: source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst (hunk 3 above)
Comment (jnasselle) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst:18]: I don't get this. What are you referring to?

File: source/installation-guide/wazuh-agent/wazuh-agent-package-solaris.rst [hunk 4]
```diff
@@ -10,7 +10,9 @@ Installing Wazuh agents on Solaris endpoints
 
 The agent runs on the host you want to monitor and communicates with the Wazuh manager, sending data in near real-time through an encrypted and authenticated channel. 
 
-To start the installation process, select your architecture: i386 or Sparc. Alternatively, if you want to download the Wazuh agent package directly, see the :doc:`packages list </installation-guide/packages-list>` section. 
+To start the installation process, select your architecture: i386 or Sparc.
```
[the remark this answers is not in the document]
Comment (jnasselle) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-solaris.rst:13]: Agree!

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst (hunk 2 above)
Comment (rauldpm) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:24]: Ok, we will delegate it to the @wazuh/content review

File: source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst (hunk 3 above)
Comment (rauldpm) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst:18]: In AIX deployment the root note is not specified either

File: source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst (hunk 1 above)
Comment (rauldpm) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-hpux.rst:16]: In other deployments it is specified before starting the process, we must be consistent with the rest of the documentation, if you consider that it should change, open an issue so the @wazuh/content can work on it.

image

File: source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst (hunk 3 above)
Comment (rauldpm) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst:18]: I'm sorry, I got confused with the file path when doing the revision

File: source/installation-guide/wazuh-agent/wazuh-agent-package-solaris.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (jnasselle) [reply source/installation-guide/wazuh-agent/wazuh-agent-package-solaris.rst]: Done in 2afd918529f3fbf2a76d8c7156427639ba1fa090

**Review by c-bordon — APPROVED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

**Review by javimed — APPROVED** [review]
LGTM

**Pull request merged**